src/OmniSharp.LanguageServerProtocol/Handlers/OmniSharpCompletionHandler.cs

```diff
@@ -112,7 +112,7 @@ private static void VerifyEnumsInSync(Type enum1, Type enum2)
             Debug.Assert(lspValues.Length == modelValues.Length);
             for (int i = 0; i < lspValues.Length; i++)
             {
-                Debug.Assert((int?)lspValues.GetValue(i) == (int?)modelValues.GetValue(i));
+                Debug.Assert((int)lspValues.GetValue(i) == (int)modelValues.GetValue(i));
```

Comment:
I changed this to `int?` because I noticed on the .NET 5.0 branch that it reports `CS8605 "Unboxing possibly null value"` there

Reply:
Well, if it's int? the assert fails because these are ints, not nullable ints :)

Reply:
oh 😅